test/Helper/GravatarTest.php

```diff
@@ -285,4 +285,12 @@ public function testEmailIsProperlyNormalized()
             $this->helper->__invoke('Example@Example.com ')->getEmail()
         );
     }
+
+    /**
+     * @expectedException PHPUnit\Framework\Error\Deprecated
```

Comment:
Please look at Travis:

> 1) ZendTest\View\Helper\GravatarTest::testSetAttribsIsDeprecated
ReflectionException: Class PHPUnit\Framework\Error\Deprecated does not exist

https://travis-ci.org/zendframework/zend-view/jobs/275727410#L730

Reply:
Yes. I forgot to comment about it (0:01 AM). I'll fix it right now

Reply:
I would suggest using method instead of annotation. I know here it doesn't matter because test contain just one call, but I would change it for consistency.

Reply:
I don't like annotations, too. But, if I remember, this use is recommended. But I'll change it, too.